pkg/generate/code/resource_reference_test.go

```diff
@@ -182,15 +182,15 @@ func Test_ReferenceFieldsPresent_NestedSliceOfStructsReference(t *testing.T) {
 	require.NotNil(crd)
 	expected :=
 		`if ko.Spec.Routes != nil {
-	for _, iter32 := range ko.Spec.Routes {
-		if iter32.GatewayRef != nil {
+	for _, iter35 := range ko.Spec.Routes {
+		if iter35.GatewayRef != nil {
```

Comment:
Oh interesting. Did this start failing just because the `api-2.json` was super out of date?

Reply:
Yup, some fields were added to `CreateRoute`:

![image](https://user-images.githubusercontent.com/61433408/167484530-91c50636-536e-418c-92ab-fd110be1a685.png)

Reply:
Oh cool cool cool yep